docs-source/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -69,6 +69,16 @@ Servers configured as `ALWAYS` count toward the cluster's `replicas` count.
 If more servers are configured as `ALWAYS` than the cluster's `replicas` count, they will all be started and the `replicas` count will be ignored.
 {{% /notice %}}
 
+### Server start state
+
+For some use cases, such as an externally managed zero downtime patching (ZDP), it may be necessary to start WebLogic
+only as far as the administrative state.  This can be achieved using the `serverStartState` property, which is available at domain,
```

Comment:
only as far as the administrative state -> (this is just a re-wording _suggestion_) so that at the end of its startup cycle, the server is in an administrative state.